include/irr/core/alloc/address_allocator_traits.h

```diff
@@ -156,9 +156,9 @@ namespace core
 
             template<class U> struct has_func_get_real_addr<U,void_t<func_get_real_addr<U> > >  : std::is_same<func_get_real_addr<U>,size_type> {};
 
-            static inline constexpr bool         supportsArbitraryOrderFrees = resolve_supportsArbitraryOrderFrees<AddressAlloc>::value;
-            static inline constexpr uint32_t maxMultiOps                           = resolve_maxMultiOps<AddressAlloc>::value;
-            static inline constexpr bool         supportsNullBuffer                 = resolve_supportsNullBuffer<AddressAlloc>::value;
+            static constexpr bool         supportsArbitraryOrderFrees = resolve_supportsArbitraryOrderFrees<AddressAlloc>::value;
```

Comment:
are you sure MSVC is complaining here?

Because I will get linking errors in GCC if the `constexpr` is not `static inline` and I sure as hell compile with `-std=c++11`